### ISSUE TITLE
Cannot register to the forum nor send mail to 'contact us'

### ISSUE THREAD

**Issue by akujbida** [FreeCAD/FreeCAD#13183]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I fill in the registration form, submit, no feedback about success; rather, the screen comes back to the terms and then to blank registration form.
The contact us button yields 'you cannot send any more messages' though I have sent zero.
I'd like to join the forum. It appears to be functional. How to get in?
Sorry for posting here, no other way I can think of to connect to the team.
Nothing special about my machine: Chrome on new macbook air M1. Browser works fine on other sites. Definitely a forum site issue.

### Full version info

```shell
[code]
OS: macOS 13.6.3
Word size of FreeCAD: 64-bit
Version: 0.21.1.33694 (Git)
Build type: Release
Branch: (HEAD detached at 0.21.1)
Hash: f6708547a9bb3f71a4aaade12109f511a72c207c
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: C/Default (C)
Installed mods: 
  * woodworking 0.21.1.33694
  * A2plus 0.4.61
  * sheetmetal 0.4.3
  * Curves 0.6.22
[/code]
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by akujbida** [comment]
After trying several more times and guessing various new user names for myself, the account got created and I clicked the link in the email to activate, with 'successful activated' message.
However, on login attempt, I get "The submitted form was invalid. Try submitting again."
Seems a rather buggy board. I see activity today, so I'm sure login isn't broken for everyone.

Ok finally got it. There was never any confirmation of successful login nor redirection to the forum. I just opened another tab to try again and saw that the login button was gone.

I'll leave this report here in case an admin hears of repeats and thinks it's worth investigating.
